src/include/utils.h

```diff
@@ -375,6 +375,67 @@ pgagroal_backtrace(void);
 
 #endif
 
+/**
+ * Command parsing utils for pgagroal-cli and pgagroal-admin.
+ */
+
+#define COMMAND_UNKNOWN (-1)
```

Comment:
Move this to the beginning of the file

Reply:
Not used in the project anymore. Removed.